geode-core/src/main/java/org/apache/geode/cache/query/internal/OrderByComparator.java

```diff
@@ -45,78 +44,50 @@ public OrderByComparator(List<CompiledSortCriterion> orderByAttrs, ObjectType ob
   }
 
   /**
-   * Yogesh : This methods evaluates sort criteria and returns a ArrayList of Object[] arrays of
-   * evaluated criteria
+   * This method evaluates sort criteria and returns an ArrayList of Object[] arrays of the
+   * evaluated criteria.
    * 
-   * @param value
-   * @return Object[]
+   * @param value the criteria to be evaluated.
+   * @return an Object array of Object arrays of the evaluated criteria.
    */
   protected Object[] evaluateSortCriteria(Object value) {
-
     CompiledSortCriterion csc;
     Object[] array = null;
     if (orderByAttrs != null) {
       array = new Object[orderByAttrs.size()];
-      Iterator orderiter = orderByAttrs.iterator();
+      Iterator orderIter = orderByAttrs.iterator();
       int i = 0;
-      while (orderiter.hasNext()) {
-        csc = (CompiledSortCriterion) orderiter.next();
-        Object[] arr = new Object[2];
-        arr[0] = csc.evaluate(value, context);
-        arr[1] = Boolean.valueOf(csc.getCriterion());
+      while (orderIter.hasNext()) {
```

Comment:
This loop and the one below could read better with a "for each" loop.
```
for (CompiledSortCriterion csc: orderByAttrs){
```

Good refactoring overall.
+1 with or without the loop edit.